helm-frontend/src/services/getSchema.ts

```diff
@@ -11,14 +13,30 @@ export default async function getSchema(signal: AbortSignal): Promise<Schema> {
 
     return (await resp.json()) as Schema;
   } catch (error) {
-    console.log(error);
-    return {
-      adapter: [],
-      metric_groups: [],
-      metrics: [],
-      models: [],
-      perturbations: [],
-      run_groups: [],
-    } as Schema;
+    // Fallback to schema.yaml if schema.json is not found since some HELMets do not have schema.json
+    try {
+      let url = getBenchmarkEndpoint(`${getVersionBaseUrl()}/`) + `schema.yaml`;
+
+      // if suite appears before benchmark_output, we need to replace that part of the url (since it means suite appears twice)
```

Comment:
unsure if we want to make this just a hardcoded fallback for HEIM or make it more general (i.e. also support release being repeated in the benchmarkEndpoint string)

Reply:
We should require HEIM to give us the `schema.json` file.

Reply:
I'll remove this then, any other things to change before this is ready to merge?